lib/presentation/widget/myspace/my_space_viewmodel.dart

```diff
@@ -188,10 +192,12 @@ class MySpaceViewModel extends BaseViewModel {
   }
 
   void shareDocuments(List<Document> documents, {ItemSelectionType itemSelectionType = ItemSelectionType.single}) {
-    if (itemSelectionType == ItemSelectionType.single) {
+    if (_appNavigation.canPop()) {
```

Comment:
what is the purpose of it?

Reply:
I want to make sure that we only pop out the widget if its possible, and I think moving this directly inside the `void popBack()` function is better, shorter code to write. let me change this :)